articles/managed-grafana/encryption.md

```diff
@@ -28,11 +28,11 @@ Data stored in Azure Cosmos DB and Azure Database for PostgreSQL is encrypted at
 
 For more information, go to [Encryption at rest in Azure Cosmos DB](/azure/cosmos-db/database-encryption-at-rest) and [Security in Azure Database for PostgreSQL - Flexible Server](/azure/postgresql/flexible-server/concepts-security).
 
-## Server-side encryption
+## Server-side Encryption
```

Comment:
Hi @sandeepsalwan1, thank you for your suggestion. Can you tell me why you're suggesting capitalizing "encryption"? We don't use title case for titles and headings on Microsoft Learn. It may seem like the previous heading is using title case but that's because because it only contains nouns are brand names that need to be capitalized.